src/viur/core/config.py

```diff
@@ -397,7 +397,17 @@ class Security(ConfigType):
 
     password_recovery_key_length: int = 42
     """Length of the Password recovery key"""
-
+    closed_system: bool = False
+
+    closed_system_allowed_paths: list[str] = [
+        "",
+        "vi",
+        "settings",
+        "skey",
+        "user/auth_googleaccount/login",
+        "user/auth_userpassword/login",
+        "user/getAuthMethods"
+    ]
```

Comment:
I would like to prefer full allow-list, with optional wildcard feature, like this.
```suggestion
    closed_system_allowed_paths: t.Iterable[str] = (
        "",  # index site
        "json/skey",
        "json/user/auth_*",
        "json/user/getAuthMethods",
        "vi/settings",
        "vi/skey",
        "vi/user/auth_*",
        "vi/user/getAuthMethods",
    )
```

This would allow to access further functions of the `auth_*`-modules, which make also `pwrecover` possible, for example.